style.css

```diff
@@ -230,136 +515,25 @@ hr {
   }
 
   #about_myself {
-    display: grid;
-    padding: 114px 24px;
-    grid-gap: 24px;
-    background: white;
-    border-radius: 0 5rem 0 0;
-  }
```

Comment:
- [ ] Ouch! this is close but If you look closely at the "About section" in the Figma design the left block takes 40% and the right one takes the remaining 60% of the page's width however In your design they seem to be taking up the same width (50/50), so please adjust this so it looks more like the Figma design.
- Figma design:
  ![image](https://github.com/medijdir/portfolio_project/assets/104968236/19fe3a1c-4668-4b0f-84e6-8bbcc628dee6)
- Your design:
  ![image](https://github.com/medijdir/portfolio_project/assets/104968236/5136eeb6-8bb2-4918-a87f-e7b8e7f01ad3)
- [ ] Also please make sure the drop-down arrow icons are properly displayed as I'm seeing random characters on my end, maybe you can consider using the icons from Font Awesome which you can find [here](https://fontawesome.com/) as they ensure consistent design regardless of the client's machine. 😉
- Your design:
  ![image](https://github.com/medijdir/portfolio_project/assets/104968236/2d5aa017-026c-42cd-a5f5-f9f2f358add4)
- Figma design:
  ![image](https://github.com/medijdir/portfolio_project/assets/104968236/99ad1d81-96a4-4910-a2b6-91edede1a5cd)